ftva_lab_data/views.py

```diff
@@ -57,3 +62,54 @@ def view_item(request, item_id):
     item = SheetImport.objects.get(id=item_id)
 
     return render(request, "view_item.html", {"item": item})
+
+
+def view_items(request: HttpRequest) -> HttpResponse:
+    return render(request, "view_items.html", context={"columns": COLUMNS})
+
+
+def render_table(request: HttpRequest) -> HttpResponse:
```

Comment:
I'd probably rename this too, to be more specific.  `render_search_results` or `render_search_table`, probably.